src/utils/ucc_parser.c

```diff
@@ -316,20 +316,24 @@ static ucc_status_t ucc_apply_file_cfg(void *opts, ucc_config_field_t *fields,
     return UCC_OK;
 }
 
-ucc_status_t ucc_config_parser_fill_opts(void *opts, ucc_config_field_t *fields,
-                                         const char *env_prefix,
-                                         const char *table_prefix,
-                                         int         ignore_errors)
+ucc_status_t ucc_config_parser_fill_opts(void *opts, ucs_config_global_list_entry_t *entry,
+                                         const char *env_prefix, int ignore_errors)
 {
     ucs_status_t ucs_status;
     ucc_status_t status;
 
-    ucs_status = ucs_config_parser_fill_opts(opts, fields, env_prefix,
-                                             table_prefix, ignore_errors);
-    status     = ucs_status_to_ucc_status(ucs_status);
+#if UCS_HAVE_CONFIG_GLOBAL_LIST_ENTRY_FLAGS
```

Comment:
Perhaps change to UCC macro instead of UCS

Reply:
prefix is UCS because it depends on libucs similarly to UCS_HAVE_MPOOL_PARAMS